content/courses/native-onchain-development/program-state-management.md

```diff
@@ -554,11 +554,10 @@ We're ready to build and deploy our program!
 You can test your program by submitting a transaction with the right instruction
 data. For that, feel free to use
 [this script](https://github.com/Unboxed-Software/solana-movie-client) or
-[the frontend](https://github.com/Unboxed-Software/solana-movie-frontend) we
-built in the
-[Deserialize Custom Instruction Data lesson](deserialize-custom-data). In both
-cases, make sure you copy and paste the program ID for your program into the
-appropriate area of the source code to make sure you're testing the right
+[the frontend](https://github.com/EmekaManuel/movie-review-dapp) we built in the
```

Comment:
Thanks for your work here. I've cloned the repo to the `solana-developers` organization (where Solana Devrel does it's work). 

```suggestion
[the frontend](https://github.com/solana-developers/movie-review-dapp) we built in the
```

Reply:
> Thanks for your work here. I've cloned the repo (including all branches) to `movie-review-frontend` under the `solana-developers` organization (where Solana Devrel does it's work) and invited you as a collaborator.
> 
> Please update all the URLs to match!

updated all URLs to match